underactuated.html

```diff
@@ -622,20 +631,19 @@
 full row rank, is dim$[\bu] < $ dim$[\bq]$.  This is the case when a robot has
 joints with no motors. But this is not the only case.  The human body, for
 instance, has an incredible number of actuators (muscles), and in many cases has
-multiple muscles per joint; despite having more actuators that position
+multiple muscles per joint; despite having more actuators than position
 variables, when I jump through the air, there is no combination of muscle inputs
 that can change the ballistic trajectory of my center of mass (barring
-aerodynamic effects).  My control system is underactuated.</p>
+aerodynamic effects). I am underactuated.</p>
```

Comment:
i prefer to talk about the control system being underactuated (and don't want to change control system to dynamical system throughout).  There are some cases where the same mecnhanical system may be underactuated or not depending on which control inputs are being used.